lib/Rx/Observable/BaseObservable.php

```diff
@@ -429,4 +454,192 @@ function() use (&$map, $observer) {
             return $refCountDisposable;
         });
     }
+
+    /**
+     * @param $value
+     * @return \Rx\Observable\AnonymousObservable
+     */
+    public static function just($value)
```

Comment:
Does it make sense to keep `just` or should we use `ReturnObservable` everywhere?